railties/lib/rails/test_unit/runner.rb

```diff
@@ -99,6 +101,14 @@ def list_tests(argv)
             tests.exclude(default_test_exclude_glob) if patterns.empty?
             tests
           end
+
+          def escape_declarative_test_filter(filter)
+            if filter.is_a?(String) && /\s+/.match?(filter)
```

Comment:
Perhaps:

```suggestion
            if filter.is_a?(String) && !filter.start_with?("test_")
```

For cases like:

https://github.com/rails/rails/blob/aaa64687e81873b26c964fada0b969621ae8cf8f/activerecord/test/cases/attribute_methods_test.rb#L68

Reply:
Excellent suggestion! :smiley: